ovos_PHAL/service.py

```diff
@@ -73,9 +73,18 @@ def load_plugins(self):
                 continue
 
             config = self.user_config.get(name) or {}
+            if config.get("enabled") is False:
+                # Configuration explicitly disabled plugin
+                LOG.debug(f"PHAL plugin {name} disabled in configuration")
+                continue
```

Comment:
I may be missing something here, but if the config value `enabled` is explicitly set to `False`, won't this leave the `enabled` variable unbound in this method? Enabled isn't defined before this point and it's defined in all of the other previous paths.

Reply:
Yes, but `continue` here means we just move on to the next item in the list, so we won't reach any references to `enabled`